internal/pkg/template/templates/environment/cf.yml

```diff
@@ -56,8 +56,93 @@ Conditions:
   ManagedAliases: !And
     - !Condition DelegateDNS
     - !Not [!Equals [ !Ref Aliases, "" ]]
+{{- if and .PublicHTTPConfig.ELBAccessLogs (not .PublicHTTPConfig.ELBAccessLogs.BucketName) }} #.PublicHTTPConfig.ELBAccessLogs.CreateBucket
+Mappings:
+  RegionalConfigs:
+    us-east-1:
+      ELBAccountId: '127311923021'
+      ArnPrefix: 'arn:aws:'
```

Comment:
Ugh this massive mappings makes me so sad as it will be difficult to maintain, what do you think of moving the `RegionalConfigs` section into a partial called `mappings-regional-configs.yml` and include it here?

Dropping a comment to the table would also be helpful: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-logging-bucket-permissions

Reply:
Also it looks like we don't need the `ArnPrefix` key

Reply:
Sure, I had these 2 actions in mind but somehow it skipped through my mind. I will change it.